src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -389,6 +393,26 @@ class IpexCommunicationService extends AgentService {
     });
   }
 
+  @OnlineOnly
+  async handleMessageFromMultisigExn(id: string) {
+    const notifRecord = await this.getNotificationRecordById(id);
+    const exn = await this.props.signifyClient
+      .exchanges()
+      .get(notifRecord.a.d as string);
+    const route = exn?.exn?.e?.exn?.r;
+
+    switch (route) {
+    case ExchangeRoute.IpexAdmit:
+      await this.acceptAcdcFromMultisigExn(id);
```

Comment:
hmm this conflicts with the other PR. We will only ever accept ACDC from a grant notification with linkedGroupRequests now

Reply:
When the PR: https://github.com/cardano-foundation/cf-identity-wallet/pull/657 is merged, I'll pull and update it.